index.js

```diff
@@ -121,6 +124,18 @@ function getTitle (html) {
   return normalize(html.slice(i, end))
 }
 
+function getFavicon (html, url) {
+  const m = html.match(/<link\s+[^>]*rel=["']?(icon|shortcut icon|apple-touch-icon)["']?[^>]*>/i)
+
+  if (m) {
+    const href = m[0].match(/href=["']([^"']+)/)
+    if (href) return href[1]
+  }
+
+  const rootUrl = new URL('/', url).href
```

Comment:
'/favicon.ico' instead of /, then you dont need the + below

Reply:
Fixed!